### PR TITLE
Sphinx - Corrections for Ensuring Complete Doc Compilation

### PR DESCRIPTION
**Pull request by mail4umar**
Changes include:

**Benchmarks**
- Updated graphs to include those ML tools that failed
- Updated Titles to include the word "vs"
- Added tabs for custom and default parameters for XGBoost
- Fixed some wrong numbers
- Changed colors of certain charts

**Tiles changes**
- API, Machine Learning , and SQL section now have different tiles - without icons

**Heirarchy changes**
- Query Performance is now moved under a new Tab i.e. Performance
- It is no longer under Utilties

**Fixed Function doc strings**
- Updated references in docstrings
- Fixed imports and other bugs
- Added missing plotting libraries

**Others**
- Modified statistics page to correct indentations
- Fixed the image in CI/CD
- Removed drop down options for version

### CODE REVIEW

File: docs/source/connect.rst [hunk 1]
```diff
@@ -9,6 +9,9 @@ ______
 Functions
 --------------------
 
+.. important::
+
+   For a compurehensive guide to create connections, please refer to :ref:`connection`.
```
Comment (oualib) [inline docs/source/connect.rst:14]: typo: comprehensive

File: docs/source/statistics.rst [hunk 2]
```diff
@@ -27,19 +27,56 @@ Here is a drill-down bar chart summarizing the VerticaPy statistics. Each bar re
...
+..   # Example.
+..   print(gen_rst_summary_table())
+..   file_path = "statistics.rst"
+..   with open(file_path, "w") as rst_file:
+..     rst_file.write(gen_rst_summary_table())
+
+.. .. include:: ../statistics.rst
+
+
++----------------------------------------+-----------------------+--------------+
```
Comment (oualib) [inline docs/source/statistics.rst:45]: is it automatically generated or you just copy pasted?

File: verticapy/core/parsers/all.py [hunk 3]
```diff
@@ -329,7 +330,7 @@ def read_file(
 
         # Cleanup block - drop / remove objects created for this example
 
-        from verticapy.utilities import drop
+        from verticapy import drop
```
Comment (oualib) [inline verticapy/core/parsers/all.py:333]: verticapy.sql import drop

File: verticapy/core/parsers/csv.py [hunk 4]
```diff
@@ -696,7 +696,7 @@ def read_csv(
 
         # Cleanup block - drop / remove objects created for this example
 
-        from verticapy.utilities import drop
+        from verticapy import drop
```
Comment (oualib) [inline verticapy/core/parsers/csv.py:699]: verticapy.sql

**Review by oualib — CHANGES_REQUESTED** [review]

File: docs/source/statistics.rst (hunk 2 above)
Comment (mail4umar) [reply docs/source/statistics.rst:45]: no it is manual for now.

**Review by oualib — APPROVED** [review]

**Pull request merged**